fibonacci/csharp/code.cs

```diff
@@ -0,0 +1,18 @@
+﻿static int Fibonacci(int n) {
```

Comment:
I know the point isn't to make the code as terse as possible, but I noticed the C++ version appears a bit simpler without unnecessary curly braces, which aren’t needed in C# either. Would you consider expressions-bodied and pattern matching? May seem unfamiliar but have become my preference in recent code and I believe should be as efficient.

``` c#
static int Fibonacci(int n) => n switch
{
    <= 1 => n,
    _ => Fibonacci(n - 1) + Fibonacci(n - 2)
};
```

Reply:
I don't like using conditional operators in pattern matching, makes the code too confusing for people not familiar with the syntax, IMO.
But a regular switch here is still good, will change

Reply:
Actually, looking at it again, this can be simplified into a ternary operator